test/integration/info/DataverseInfoRepository.test.ts

```diff
@@ -0,0 +1,12 @@
+import { DataverseInfoRepository } from '../../../src/info/infra/repositories/DataverseInfoRepository';
+
+describe('getDataverseVersion', () => {
+  // TODO: Change API URL to another of an integration test oriented Dataverse instance
+  const sut: DataverseInfoRepository = new DataverseInfoRepository('https://demo.dataverse.org/api/v1');
```

Comment:
I think for now having the demo server here is fine but we should talk more about the longer term plan for testing.

This comes up for all client libraries! 😄

Reply:
The conversation about API testing continues in Containerization Working Group meetings: https://ct.gdcc.io